### PR TITLE
Fix missing braces

### PR DESCRIPTION
**Pull request by feraudet**

### CODE REVIEW

**Comment by mauricios** [comment]
@feraudet thanks for spotting this bug. @damaya Can you review the fix?

File: main.tf [hunk 1]
```diff
@@ -12,8 +12,8 @@ resource "aws_route53_record" "this" {
   name            = aws_acm_certificate.this.domain_validation_options[count.index]["resource_record_name"]
   type            = aws_acm_certificate.this.domain_validation_options[count.index]["resource_record_type"]
   zone_id         = element(var.zone_ids, count.index)
-  records = aws_acm_certificate.this.domain_validation_options[count.index]["resource_record_value"]
-  ttl     = 60
+  records         = [aws_acm_certificate.this.domain_validation_options[count.index]["resource_record_value"]]
```
Comment (damaya) [inline main.tf:15]: Salut @feraudet Merci pour votre contribution :)
According to the updated docs https://www.terraform.io/docs/providers/aws/r/route53_record.html the records declaration is ok with brackets. I am merging this to the master, and we'll create a new tag.
à bientôt!

**Review by damaya — APPROVED** [review]

**Comment by damaya** [comment]
@feraudet please use the module from Terraform Registry, with version 0.2.1 https://registry.terraform.io/modules/Aplyca/acm/aws/0.2.1

File: main.tf (hunk 1 above)
Comment (feraudet) [reply main.tf:15]: Merci :)

**Pull request merged**